lib/media/drm_engine.js

```diff
@@ -1185,16 +1218,18 @@ shaka.media.DrmEngine = class {
       if (!present) {
         this.activeSessions_.delete(session);
 
-        this.onError_(new shaka.util.Error(
-            shaka.util.Error.Severity.CRITICAL,
-            shaka.util.Error.Category.DRM,
-            shaka.util.Error.Code.OFFLINE_SESSION_REMOVED));
-        return Promise.resolve();
+        if (this.config_.persistentSessionOnlinePlayback) {
+          shaka.log.warning('Persistent session', sessionId, 'not available');
+        } else {
+          this.onError_(new shaka.util.Error(
+              shaka.util.Error.Severity.CRITICAL,
+              shaka.util.Error.Category.DRM,
+              shaka.util.Error.Code.OFFLINE_SESSION_REMOVED));
+
+          return Promise.resolve();
+        }
       }
 
-      // TODO: We should get a key status change event.  Remove once Chrome CDM
```

Comment:
Downloaded Widevine DRM content worked great with Chrome ;)

Reply:
What happens in old TV browsers?

Reply:
Haven't tested yet the persistent licenses on TV but will do

Reply:
### Tizen 6.5
It correctly starts the persistent-license session at first launch but fails on the session load at second launch, so my guess is that this isn't supported.
The sessionId is always the same `initialdrmsessionid` so I don't think it could work at all.
```
Shaka Error DRM.FAILED_TO_CREATE_SESSION (Failed to execute 'load' on 'MediaKeySession': Loading session failed. . (2))
```

### webOS 5.0
It fails to request a media key system access when asking for a `persistent-license` with the error `REQUESTED_KEY_SYSTEM_CONFIG_UNAVAILABLE`, so it seems it doesn't support persistent license at all


<br />

So my overall conclusion is persistent license isn't supported on TV devices, so I don't think we need to go further into making this feature working on those devices.


<br />

### Error Recoverability
It pointed me out on an issue with my code where fails to load a persistent license will stop the player, rather than trying to fetch a new one (intended behaviour).


I have two options for that, let me know what you prefer (my pref is option 1):

- Option 1: Send a RECOVERABLE error
```
this.onError_(new shaka.util.Error(
    this.config_.persistentSessionOnlinePlayback ?
      shaka.util.Error.Severity.RECOVERABLE : shaka.util.Error.Severity.CRITICAL;
    shaka.util.Error.Category.DRM,
    shaka.util.Error.Code.FAILED_TO_CREATE_SESSION,
    error.message));
```

- Option 2: Bypass the error, only log something:
```
if (this.config_.persistentSessionOnlinePlayback) {
  shaka.log.warning('Persistent session', sessionId, 'not available');
} else {
  this.onError_(new shaka.util.Error(
      shaka.util.Error.Severity.CRITICAL,
      shaka.util.Error.Category.DRM,
      shaka.util.Error.Code.OFFLINE_SESSION_REMOVED));

  return Promise.resolve();
}
```